examples/Demo/Shared/Components/SiteSettingsPanel.razor

```diff
@@ -22,8 +21,8 @@
         </FluentSwitch>
         <FluentSwitch @bind-Value="@_rtl" title="Switch direction" AfterBindValue="UpdateDirection">
             <span>Direction</span>
-            <span slot="unchecked-message">LTR</span>
-            <span slot="checked-message">RTL</span>
+            <span slot="unchecked-message">RTL</span>
```

Comment:
Why did you change this order? LTR is by default (and unchecked)

Reply:
I'll undo that 👍